### ISSUE TITLE
Compatibility with updated (?) Gliph2

### ISSUE THREAD

**Issue by prestigious-bat3868** [davidcoffey/TCellPack#2]
Hi there, I was wondering if this was still compatible with Gliph2.

Gliph2 now outputs a csv file, instead of "cluster.txt". Pointing PlotTCell to the csv file generates an error:

> > PlotTCellPack(gliph = "Data/P10290_BR7LWK1NL2.csv")
> Read 3277 items
> Error in `loc_validate()`:
> ! Can't subset columns past the end.
> ℹ Location 5 doesn't exist.
> ℹ There are only 4 columns.
> Run `rlang::last_error()` to see where the error occurred.

Thank you in advance.

**Comment by davidcoffey** [comment]
Yes, the current version of TCellPack is compatible with GLIPH2.  Ensure you use the cluster.txt file output from GLIPH2 as input into TCellPack.
